README.md

```diff
@@ -56,18 +57,39 @@ The tool has been tested with the following software versions.
 
 ### Executable Installation
 
-In order to install the command line interface, navigate to the root directory of the project and run
+In order to install the command line interface, navigate to the root directory of the project and run the following command.
 
 ```bash
 cabal new-install exe:haskell-src-transformations
 ```
 
-You can also run the executable directly without installing it first.
+The command above builds the executable pattern matching compiler's command line interface and creates a symbolic link in `~/.cabal/bin`.
+To test whether the installation was successful, make sure the `~/.cabal/bin` is in your `PATH` environment variable and run the following command.
+
+```
+haskell-src-transformations --help
+```
+
+### Running without Installation
+
+If you want to run the command line interface without installing the package on your machine (e.g., for debugging purposes), execute the following command in the root directory of the project instead of the `haskell-src-transformations`.
```

Comment:
What do you mean with "instead of the `haskell-src-transformations`"?

Reply:
I think I meant "instead of the `haskell-src-transformations` command". Whenever the user would usually run the `haskell-src-transformations` command (after installing the tool), the other command should be executed instead (in the workflow without installation). Do you have suggestions for how to improve the sentence? : )

Reply:
This would be my suggestion:
```suggestion
If you want to run the command line interface without installing the package on your machine (e.g., for debugging purposes), execute the following command in the root directory of the project instead of using the `haskell-src-transformations` executable.
```